### PR TITLE
Fix/ci pr

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: ci/install-package.sh [hunk 1]
```diff
@@ -121,25 +123,40 @@ then
...
+DOCKER_HOME=$(docker exec -t tue-env bash -c 'source ~/.bashrc; echo "$HOME"' | tr -d '\r')
 
-# Set the package to the right commit
-echo -e "\e[35m\e[1m Reset package to this commit \e[0m"
-if [[ $PULL_REQUEST == "false" ]]
+if [[ $PULL_REQUEST != "false" ]]
 then
-    echo -e "\e[35m\e[1m cd ~/ros/$ROS_DISTRO/system/src/$PACKAGE && git reset --hard $COMMIT \e[0m"
-    docker exec -t tue-env bash -c 'source ~/.bashrc; cd ~/ros/"$ROS_DISTRO"/system/src/"$PACKAGE" && git reset --hard "$COMMIT"'
+    # Install only the git repo of the package
```
Comment (MatthijsBurgh) [inline ci/install-package.sh:138]: Add more comments for later me.

File: ci/.travis.yml.multiple_pkgs.example [hunk 2]
```diff
@@ -22,7 +23,10 @@ env:
   - PACKAGE=package3
 
 install:
-  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST --image="tuerobotics/tue-env"
+  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST
 
 script:
   - bash build-package.sh --package=$PACKAGE
```
Comment (MatthijsBurgh) [inline ci/.travis.yml.multiple_pkgs.example:29]: Build is going to  be in `before_script`, test in `script`

File: ci/.travis.yml.example [hunk 3]
```diff
@@ -8,10 +8,14 @@ services:
 before_install:
   - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
   - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/test-package.sh
   - export PACKAGE=${TRAVIS_REPO_SLUG#*/}
 
 install:
-  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST --image="tuerobotics/tue-env"
+  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST
```
Comment (MatthijsBurgh) [inline ci/.travis.yml.example:15]: --branch should always be TRAVIS_BRANCH

**Comment by ar13pit** [comment]
This looks good to me. Lets hold on to this for a while till we figure out a good way of fixing all the CI configs.

**Comment by ar13pit** [comment]
Close #515 and Close #516

**Pull request merged**